i2c/26-i2c.js

```diff
@@ -43,9 +43,10 @@ module.exports = function(RED) {
         node.on("input", function(msg) {
             var address = node.address || msg.address ;
             var command = node.command || msg.command ;
+            var buffcount = node.count || msg.bytes ;
             address = parseInt(address);
             command = parseInt(command);
-            var buffcount = parseInt(node.count);
+            buffcount = parseInt(buffcount);
```

Comment:
need to ensure we always get a number (if using msg.bytes) - or raise an error.

Reply:
this is already done a few lines later (if statement at line 53)

Reply:
ah yes - ok.